dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -142,17 +145,24 @@ protected static class TeradataLogsJdbcTask extends AbstractJdbcTask<Void> {
         protected final SharedState state;
         protected final String logTable;
         protected final String queryTable;
-        protected final String condition;
+        protected final List<String> conditions;
         protected final ZonedInterval interval;
+        protected final List<String> orderBy;
 
         public TeradataLogsJdbcTask(@Nonnull String targetPath, SharedState state, String logTable,
-            String queryTable, String condition, ZonedInterval interval) {
+                                    String queryTable, List<String> conditions, ZonedInterval interval) {
+            this(targetPath, state, logTable, queryTable, conditions, interval, Collections.emptyList());
+        }
+
+        public TeradataLogsJdbcTask(@Nonnull String targetPath, SharedState state, String logTable,
+                                    String queryTable, List<String> conditions, ZonedInterval interval, List<String> orderBy) {
```

Comment:
Can we make this new constructor protected so that nobody except the assessment task can construct a log dumper with an order? I'm very keen to make sure that we don't accidentally lose the fast-path.

Reply:
Done